erroranalysis/tests/test_cohort_filter.py

```diff
@@ -137,6 +137,40 @@ def test_cohort_filter_in_the_range_of(self):
                            model_task,
                            filters=filters)
 
+    @pytest.mark.parametrize('arg, correct_prediction',
+                             [([1], False), ([0], True)])
+    def test_cohort_filter_multiclass_classification_outcome(
+            self, arg, correct_prediction):
+        X_train, X_test, y_train, y_test, feature_names = create_iris_pandas()
+        model = create_sklearn_svm_classifier(X_train, y_train)
+        model_task = ModelTask.CLASSIFICATION
+        categorical_features = []
+
+        # the index 1, corresponds to incorrect prediction
+        # the index 0 correspond to correct prediction
+        filters = [{'arg': arg,
```

Comment:
I thought I had constants for all of these - arg, column, method

Reply:
ah yes, there is:
```
from erroranalysis._internal.constants import (ARG, COLUMN, METHOD)
```
but it looks like the other tests also need to be updated in this file to use them

Reply:
Yeah, I have create a task internally to migrate these to module constants.